### DISCUSSION TITLE
Branch scope highlighting

### DISCUSSION THREAD

**Discussion by classicrob** [vslinko/obsidian-outliner#56] [Ideas]
**Is your feature request related to a problem? Please describe.**

Sometimes with long outlines, it's hard to tell what level you are at to make sure your indentation is correct.

**Describe the solution you'd like**

Branching scope highlighting. This is a plugin that I designed for Roam during the [#roamgames onboarding challenge](https://www.figma.com/file/5shwLdUCHxSaPNEO7pazbe/Dhrumil-and-Robert-RoamGames-Challenge-2?node-id=0%3A1), and highlighting the page references doesn't really make sense in Obsidian's context, but coloring the bullets and the branch to the bullet that you're currently editing is incredibly helpful, on desktop and mobile! 
![image](https://user-images.githubusercontent.com/46608303/113022724-2506be00-9142-11eb-8404-3d07891e068a.png)

**Describe alternatives you've considered**

Zooming in and out on blocks frequently is just a bit too distracting.

**Comment by vslinko** [comment]
@classicrob Thank you for your idea! It looks really helpful.
But I think it's better to make plugin more stable and add multiline support first, so, I created milestone 1.3.0 with this feature.

**Comment by vslinko** [comment]
I've converted the issue into discussions because it's very deep in my backlog and because it's possible to vote in discussions.

**Comment by I-Pch** [comment]
This would be an amazing add-on to the plugin 😁 👍 !
